algrtm/src/components/LinksSection.js

```diff
@@ -0,0 +1,90 @@
+// TODO add proper links
+
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+import externalLinks from "@/data/externalLinks";
+
+const LinksSection = () => {
+  return (
+    <div className=" bg-neutralLight-neutral10 w-full border-t-2 border-neutralLight-neutral20">
+      <div className="flex flex-col bg-neutralLight-neutral10 mx-[12.5rem] mt-16 mb-[7.75rem] gap-16">
```

Comment:
It seems to me that the problem with the squeezing of the links in the right part of this component is due to the margins. I would use `max-w-[65.563rem]` here, and add `flex justify-center items-center pt-16 pb-[7.75rem]` to the parent. That way, this section should have the correct width